lib/core/feature-flags/src/lib/components/flags/flags.component.scss

```diff
@@ -0,0 +1,112 @@
+/* stylelint-disable selector-class-pattern */
```

Comment:
disabled because there are Material css selectors (`mat-`)
I've seen it has been used in other parts of the sourcecode